src/dil_demo/store.clj

```diff
@@ -92,3 +82,13 @@
     (process-store req
                    (app (assoc-store req config))
                    config)))
+
+(defn make-store
+  "Return a store atom loaded with data from `file` (EDN format) and
+  automatically stored to that file on changes."
+  [{:keys [file]}]
+  (let [store (atom (load-store file))]
+    (add-watch store nil
+               (fn [_ _ old-store new-store]
+                 (when (not= old-store new-store)
+                   (future (save-store new-store file)))))))
```

Comment:
oh ik zie dit nu pas, maar `future` geeft geen garanties over wanneer de body uitgevoerd wordt; dus latere `futures` kunnen eerder dan eerdere `futures` uitgevoerd worden. of tegelijk....